### DISCUSSION TITLE
Issues in extending login page

### DISCUSSION THREAD

**Discussion by lsatolli** [filamentphp/filament#9489] [Help] [answered]
Accordingly to the documentation of filament:
https://filamentphp.com/docs/3.x/panels/users#customizing-the-authentication-features

and to this case studies:
https://laraveldaily.com/post/filament-3-login-with-name-username-or-email

I've tried to extend login page. I tried to change the view associated to the class and/or tried to follow the use case of the second link changing the name of the "email" field in login page:

![immagine](https://github.com/filamentphp/filament/assets/79750699/31e1fbe6-dd8b-4377-9fbb-f8e4e75b14de)

I've correctly changed the adminProvider.php file 
![immagine](https://github.com/filamentphp/filament/assets/79750699/548e4861-47de-4905-b0f1-d4c2813f536c)

but all the changes I made are ignored and the login form is still the default one:

![immagine](https://github.com/filamentphp/filament/assets/79750699/d73ef3af-080b-4d39-8f94-294fd97851cb)

**Comment by VincvanAs** [comment, marked as answer]
Change Filament\Pages\Auth\Login;
To App\Filament\Pages\Auth\Login; in your panel provider.

**Comment by iamjothees** [comment]
In my case
`
$panel->plugin(new \BetterFuturesStudio\FilamentLocalLogins\LocalLogins\LocalLogins())
`
overrided the `->login()`
